src/app/create-suggested-dates/create-suggested-dates.component.html

```diff
@@ -29,20 +29,24 @@
         [attr.aria-readonly]="isSuggestedDateFromDatabase(i)" [formGroup]="castToFormGroup(suggestedDateForm)"
         class="section-container pb-2">
         <!-- title of the container -->
-        <div class="section-title text-center p-3">
-          <span class="float-end">
+        <div class="section-title text-center p-3 position-relative">
+          <h2 class="font-medium text-bold d-inline" data-id="enterDate">
+            {{ 'date.enter' | translate }} {{ i + 1 }}
+          </h2>
+          <span class="position-absolute">
             <button
               (click)="deleteSuggestedDateFormByIndex(i)"
               class="btn btn-transparent p-0 border-0 btn-delete-date"
               id="removeDate-{{i}}"
               type="button"
             >
-              <img alt="{{ 'date.delete' | translate }}" class="icon" ngSrc="assets/delete-date.svg" height="height"
-                   width="width"/>
+              <img
+                alt="{{ 'date.delete' | translate }} {{ i + 1 }}"
```

Comment:
wäre es nicht besser, das Datum + Uhrzeit vorzulesen? Ich mein so ist es besser als vorher, aber trotzdem noch umständlich, einen bestimmten Termin zu löschen

Reply:
Der Alt Text entspricht nun dem Titel/Label der Terminoption. Werde mir das mit Datum & Uhrzeit aber nochmal anschauen, klingt erstmal gut.

Zum Thema Liveregion: Bei einem [Issue von NVDA](https://github.com/nvaccess/nvda/issues/14505#issuecomment-1424977802) wurde geschrieben, dass fokussierte Element eh vorgelesen werden und daher auf ein aria-live verzichtet werden sollte. Dass mit diesen Änderungen VoiceOver gar nicht funktioniert ist natürlich schlecht, ich werde es entsprechend korrigieren.

Reply:
In dem verlinkten Issue geht es aber um einen Button, der ja eh fokussiert wird.

Unser Problem ist doch aber, dass Fehlermeldungen nie fokussiert und damit auch nicht vorgelesen werden. Bei uns sind die mit einem aria-describedby verlinkt, so dass sie vorgelesen werden, sobald das Input-Feld den Fokus bekommt. Wenn aber eine Fehlermeldung erst unter dem Input auftaucht, wenn man das Feld verlässt, wird das auch nicht mehr vorgelesen. Oder manche Fehlermeldungen beziehen sich ja auch auf mehrere Inputs, z.B. die Prüfung dass das Startdatum vor dem Endedatum liegen muss.
So wie ich es verstehe, ist aria-live für genau solche Szenarien gedacht, in denen sich etwas in einem Bereich ändert, der außerhalb des aktuellen Fokus liegt: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-live.

Tatsächlich falsch war es allerdings wirklich, auch das Input selbst zum Teil der Live-Region zu machen, da hast du Recht. Ich würde nur vorschlagen, die Region zu verkleinern so dass sie nur die Fehlermeldungen umfasst.

Lass uns dazu im Zweifel nochmal telefonieren.